apps/web/lib/features/auth-user-task-input.tsx

```diff
@@ -23,40 +23,39 @@ export function AuthUserTaskInput({ className }: IClassName) {
 				showTaskNumber={true}
 				autoAssignTaskAuth={isTrackingEnabled}
 			/>
-
-			<div className="flex flex-col justify-between gap-5 lg:flex-row lg:items-center">
-				<div className="flex mb-4 lg:mb-0">
-					<span className="font-normal text-gray-500">{t('common.ESTIMATE')}:</span>
+			<div className="flex gap-5 flex-row lg:items-center justify-between ml-2">
+				<div className="xl:flex mb-4  lg:mb-0">
+					<span className="font-normal  text-gray-500 pr-2">{t('common.ESTIMATE')}:</span>
 					<TaskEstimate />
 				</div>
 
-				<div className="justify-end flex-1 hidden gap-2 md:flex">
```

Comment:
@Anishali2 can it be that "hidden" was here on purpose to hide those elements initially and show only when needed? I am not sure, just asking?

Reply:
Not sure because I don't change that hide and show css it's default but I'll check and fix that

Reply:
@Anishali2 I think you forgot to check / fix that, I put back that "hidden" class, hope it makes sense?